backend-opal/main.js

```diff
@@ -1,5 +1,14 @@
 require('dotenv').config();
+const express = require('express');
+const portfolio = require('./src/server/portfolio');
 
-const initServer = require('./src/server/init');
+const server = express();
 
-initServer();
+// wait for handlers
+portfolio.getPortfolio(server);
```

Comment:
@WenyuD instead of this solution let's use the official one:
https://expressjs.com/en/guide/routing.html#express-router